embetter/text/_word2vec.py

```diff
@@ -0,0 +1,140 @@
+from typing import Literal, Union
+
+import numpy as np
+from gensim import downloader
+from gensim.models import KeyedVectors, Word2Vec
+from gensim.utils import tokenize
+
+from embetter.base import EmbetterBase
+
+
+class Word2VecEncoder(EmbetterBase):
```

Comment:
Might be nice to add a unit test for this. It could even be a test that takes a small text file, [trains embeddings for it](https://radimrehurek.com/gensim/models/word2vec.html#usage-examples), saves it to a tmpdir and then we confirm that it can load it.

Reply:
Actually ... does this support the keyedvectors?